### PR TITLE
VeranikaCharlotte1

### PR DESCRIPTION
**Pull request by VeranikaCharlotte**
testH2TagText_WhenSearchingCityCountry() added

### CODE REVIEW

**Review by DenSebrovsky — APPROVED** [review]
Please, delete empty rows - 42, 43, 45

File: src/test/java/VeranikaSukhikhTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+public class VeranikaSukhikhTest extends BaseTest {
+    @Test
+    public void testH2TagText_WhenSearchingCityCountry() throws InterruptedException {
+
+        String url = "https://openweathermap.org/";
+        String cityName = "Paris";
+        String expectedResult = "Paris, FR";
+
+        getDriver().get(url);
+        Thread.sleep(5000);
```
Comment (Rom-T) [inline src/test/java/VeranikaSukhikhTest.java:16]: увеличьте sleep до 10 сек

**Review by Rom-T — APPROVED** [review]

File: src/test/java/VeranikaSukhikhTest.java (hunk 1 above)
Comment (VeranikaCharlotte) [reply src/test/java/VeranikaSukhikhTest.java:16]: thank you

File: src/test/java/VeranikaSukhikhTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                By.xpath("//div[@id = 'weather-widget']//input[@placeholder = 'Search city']")
+        );
+        searchCityField.click();
+        searchCityField.sendKeys(cityName);
+
+        WebElement searchButton = getDriver().findElement(
+                By.xpath("//div[@id = 'weather-widget']//button[@type = 'submit']")
+        );
+        searchButton.click();
+        Thread.sleep(1000);
```
[the remark this answers is not in the document]
Comment (VeranikaCharlotte) [reply src/test/java/VeranikaSukhikhTest.java:28]: thank you

File: src/test/java/VeranikaSukhikhTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        WebElement parisFRChoiceInDropdownMenu = getDriver().findElement(
+                By.xpath("//ul[@class = 'search-dropdown-menu']/li/span[text() = 'Paris, FR ']")
+        );
+        parisFRChoiceInDropdownMenu.click();
+
+        WebElement h2CityCountryHeader = getDriver().findElement(
+                By.xpath("//div[@id = 'weather-widget']//h2")
+        );
+        Thread.sleep(2000);
```
[the remark this answers is not in the document]
Comment (VeranikaCharlotte) [reply src/test/java/VeranikaSukhikhTest.java:38]: thank you

**Pull request closed without merging**